src/main/kotlin/com/autonomousapps/internal/utils/gradleStrings.kt

```diff
@@ -200,12 +200,7 @@ internal fun Dependency.toCoordinates(): Coordinates? {
  */
 internal fun Dependency.toIdentifier(): Pair<ModuleInfo, GradleVariantIdentification>? = when (this) {
   is ProjectDependency -> {
-    val identifier = if (GradleVersions.isAtLeastGradle811) {
-      path
-    } else {
-      @Suppress("DEPRECATION")
-      dependencyProject.path
-    }
+    val identifier = path
```

Comment:
This change and similar mean the minimum version of Gradle we can support is now 8.11.